app/src/androidTest/java/com/andela/art/login/LoginActivityTest.java

```diff
@@ -62,6 +58,7 @@ public void loginActivityTest() {
                                 1),
                         isDisplayed()));
 //        googleSignInButton.perform(click());
```

Comment:
Clicking the button does add coverage because more of the code is invoked after clicking the button. What tests are failing, commenting out the code seems like a shortcut no?

Reply:
If you pull this branch and then run all the tests you will observe that the dialog is not dismissed which causes some activities not to launch. Also the point of testing is not adding coverage, it's ensuring a certain functionality works. The test does not check any functionality. I found no way of dismissing the dialog.

Reply:
By clicking the sign in button it launches the Intent from `GoogleSignInApi.getSignInIntent`. So that dialog actually showing up does actually mean that our implementation of the `GoogleSignInClient` is okay. So I find it necessary to maintain it as part of the test. All tests run fine locally and on my PR, what tests are not working for you?

Reply:
Okay, let me have a look into this and find a way to dismiss the dialog.